modules/installation-and-upgrade/pages/hardware-requirements.adoc

```diff
@@ -124,11 +124,18 @@ Double the space if the server is an ISS Master.
 
 |===
 
+endif::[]
 
 
-endif::[]
+[IMPORTANT]
+====
+{productname} performance depends on a number of underlying resources, for example network bandwidth and latency between clients and servers.
 
+Based on the experience and different deployments that are in use, the advice for optimal performance of {productname} Server  with an adequate number of proxies is to not exceed 10 000 clients per single server.
+It is strongly recommended to move to Hub setup and involve consultancy once you have more than 10 000 clients, as this can lead to performance issues.
```

Comment:
```suggestion
It is highly recommended to move to the Hub setup and involve consultancy once you have more than 10 000 clients, as this can lead to performance issues.
```